plugin/trino-hive/pom.xml

```diff
@@ -531,6 +531,7 @@
                             <excludes>
                                 <exclude>**/TestHiveGlueMetastore.java</exclude>
                                 <exclude>**/TestTrinoS3FileSystemAwsS3.java</exclude>
+                                <exclude>**/TestS3SelectPushdown.java</exclude>
```

Comment:
This is excluding already existing tests due to conflict file naming, please see my comment below.